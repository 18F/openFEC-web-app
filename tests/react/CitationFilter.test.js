const rendered = require('react-test-renderer');
const React = require('react');
const ReactDOM = require('react-dom');
const CitationFilter = require('../../static/js/legal/filters/CitationFilter');

it('renders checkbox list', () => {
  const component = rendered.create(
    <CitationFilter handleChange={() => {}} getResults={() => {}}
        key="ao_regulatory_citation" name="ao_regulatory_citation" label="Regulatory citation"
        instantQuery={() => {}} citationType="regulation" value=""
        resultCountChange={0} lastFilter=""/>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
