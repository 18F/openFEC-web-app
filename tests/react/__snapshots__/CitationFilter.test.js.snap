// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders checkbox list 1`] = `
<div
  className="filter"
  onBlur={[Function]}
>
  <label
    className="label"
    htmlFor="ao_regulatory_citation-filter"
  >
    Regulatory citation
  </label>
  
  <div
    className="combo combo--search--mini"
    style={
      Object {
        "display": "block",
        "position": "relative",
      }
    }
  >
    <input
      className="combo__input"
      id="ao_regulatory_citation-filter"
      name="ao_regulatory_citation"
      onChange={[Function]}
      type="text"
      value=""
    />
    <div
      aria-live="polite"
      className="tt-menu"
      style={
        Object {
          "display": "none",
          "left": "0px",
          "position": "absolute",
          "top": "100%",
          "zIndex": 100,
        }
      }
    >
      <div
        className="tt-dataset tt-dataset-candidate"
      >
        <span
          className="tt-suggestion__header"
        >
          Select a citation:
        </span>
      </div>
    </div>
  </div>
</div>
`;
